src/worker.js

```diff
@@ -149,6 +163,7 @@ this.onmessage = function(e) {
       if (typeof FS !== 'undefined' && typeof FS.createStandardStreams === 'function') FS.createStandardStreams();
 #endif
       postMessage({ cmd: 'loaded' });
+#endif
```

Comment:
please add a comment `// MODULARIZE && EXPORT_ES6` so it's easier to match up the if and its end